hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFsSnapshot.java

```diff
@@ -285,6 +285,19 @@ public void testDeleteBucketWithSnapshot() throws Exception {
     Assertions.assertTrue(listSnapKeyOut.contains(snapshotKeyPath));
   }
 
+  @Test
+  public void testSnapshotDeleteSuccessAndFailure() throws Exception {
```

Comment:
nit: I prefer to test only one thing in a test. Also by making `deleteSnapshot()` another function you are delegating test to another function. Better way would have been to have two different test one for success and one for failure.

Reply:
Understood. The test was pretty small so added it to the same test.